README.md

```diff
@@ -12,46 +12,58 @@ oc process -f https://raw.githubusercontent.com/jaegertracing/jaeger-openshift/m
 Once everything is ready, `oc status` tells you where to find Jaeger URL.
 
 ## Production setup
-This template deploys all Jaeger components as a separate services: StatefulSet Cassandra storage, agent as DaemonSet,
-collector and query service with UI. Each one of those can be managed and scaled individually. Because this template
-deploys the agent as a DaemonSet it requires system:admin permissions, therefore cannot be used in OpenShift online.
+This template deploys all Jaeger components as a separate services: StatefulSet Cassandra storage and a service for the Collector
+and for the Query with UI. Each one of those can be managed and scaled individually.
 
 Install everything in `jaeger` namespace:
```

Comment:
It says in "jaeger" namespace, but command below has been updated to use `jaeger-infra`